src/pages/Open-source-M1/index.md

```diff
@@ -7,15 +7,15 @@ date: "2018-09-20T23:46:37.121Z"
 #### How to start with Open-Source?
 #### How to step into Open-Source Development?
 
-These are some simple yet daunting questions that many people have in mind. Some question up while the rest don't even care to understand it. And so, in order to remove this confusion or dilemma of the community members, a meetup was organized on the agenda 
-"How to Open Source!" on 14th September 2018.
+These are some simple yet daunting questions that many people have in mind. Some question up while the rest don't even care to understand it. And so, in order to remove this confusion or dilemma of the community members, a meetup was organized on the agenda "How to Open Source!" on 14th September 2018.
 
 An integral part of organizing meetups is to make the community members interact and network with each other. So, the meetup started with an introduction part where participants had to introduce themselves and the technologies they were working on/interested to work on.
 
+
```

Comment:
@saranshbarua I forgot to comment this. @weirdwiz why do we still have an extra newline?

Reply:
I deleted my fork a while ago, now I can't edit the files. Would it work if i fork again?

Reply:
I... don't think so.

This PR still exists though, so if I were you, I would `curl` the patch file (https://patch-diff.githubusercontent.com/raw/osdc/blog/pull/14.patch) and `git am` to apply the patch.

Reply:
The PR exists but the repository doesn't exist to which I would push to, maybe I could close this PR and make another one?